lib/registrant.rb

```diff
@@ -0,0 +1,21 @@
+class Registrant
+attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, we don't want to expose methods we don't need. `permit` is not a method we need. We already have `permit?`